src/main/java/mmosii/bookstore/model/CartItem.java

```diff
@@ -0,0 +1,33 @@
+package mmosii.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
```

Comment:
be careful with data - it will also include shopping cart and book, which are supposed to be lazy